matRad_calcParticleDoseBixel.m

```diff
@@ -23,25 +23,21 @@
 
 % %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %
-% Copyright 2015, Mark Bangert, on behalf of the matRad development team
+% Copyright 2016, Mark Bangert, on behalf of the matRad development team
```

Comment:
the help statement in line 7 (above) needs to be adjusted to reflect the changes in input arguments.